docs/releases/1.8.0.md

```diff
@@ -0,0 +1,124 @@
+* Update spotless to 2.30.0 by @snuyanzin in https://github.com/datafaker-net/datafaker/pull/628
+* Add Kotlin examples for formats in schemas.md (docs) by @panilya in https://github.com/datafaker-net/datafaker/pull/633
+* Fix broken link in the docs by @panilya in https://github.com/datafaker-net/datafaker/pull/630
+* Introduce singletone Locale to use identity hash maps by @snuyanzin in https://github.com/datafaker-net/datafaker/pull/631
+* #635 - Expand SV local names by @robosoul in https://github.com/datafaker-net/datafaker/pull/638
+* #637 - Expand DK local names by @robosoul in https://github.com/datafaker-net/datafaker/pull/640
+* #636 - Expand NO local names by @robosoul in https://github.com/datafaker-net/datafaker/pull/639
+* Docu fix by @bodiam in https://github.com/datafaker-net/datafaker/pull/634
+* Enhance NL local company name and suffix by @robosoul in https://github.com/datafaker-net/datafaker/pull/641
+* #642 - Language code for Denmark is 'da' by @robosoul in https://github.com/datafaker-net/datafaker/pull/643
+* #644 - reusable merge of male/female names in the first_name collection by @robosoul in https://github.com/datafaker-net/datafaker/pull/645
+* Use https for schemaLocation by @snuyanzin in https://github.com/datafaker-net/datafaker/pull/650
+* Provider groups description and new providers per version table in DOCS by @panilya in https://github.com/datafaker-net/datafaker/pull/649
+* Computer brands feature request by @HattoriHenzo in https://github.com/datafaker-net/datafaker/pull/654
+* :technologist: JsonTransformer enhancement to support import scenarios by @mesketh in https://github.com/datafaker-net/datafaker/pull/653
+* Make properties ordered by @snuyanzin in https://github.com/datafaker-net/datafaker/pull/655
+* Add configuration for comma in json, refactor jsontransformer by @snuyanzin in https://github.com/datafaker-net/datafaker/pull/656
+* Garment Size Test Tweak by @kingthorin in https://github.com/datafaker-net/datafaker/pull/657
+* Add Zodiac Signs provider by @kingthorin in https://github.com/datafaker-net/datafaker/pull/658
+* Rename 'Show' group to 'Entertainment' and add new provider by @panilya in https://github.com/datafaker-net/datafaker/pull/647
+* Enforce EOL=LF by @kingthorin in https://github.com/datafaker-net/datafaker/pull/659
+* Correct custom provider example by @AlexJFerreira in https://github.com/datafaker-net/datafaker/pull/661
+* Improve unit testing by @kingthorin in https://github.com/datafaker-net/datafaker/pull/660
+* Remove usage Format Json from BaseFaker by @RVRhub in https://github.com/datafaker-net/datafaker/pull/663
+* Include XML files in .gitattributes as text by @kingthorin in https://github.com/datafaker-net/datafaker/pull/664
+* Add computer brands by @kingthorin in https://github.com/datafaker-net/datafaker/pull/665
+* implement currently passing Xlint checks by @kingthorin in https://github.com/datafaker-net/datafaker/pull/667
```

Comment:
We should drop this line for now :wink:

Reply:
well, wellllll... I just hope it will come back soonish!